packages/cli/src/generators/app.ts

```diff
@@ -42,16 +42,23 @@ class AppGenerator extends Generator<AppGeneratorOptions> {
   }
 
   async postWrite() {
-    const pkg = readJSONSync(join(this.destinationPath(), 'package.json'))
+    const pkgJsonLocation = join(this.destinationPath(), 'package.json')
+    const pkg = readJSONSync(pkgJsonLocation)
     const pkgDependencies = Object.keys(pkg.dependencies)
     const pkgDevDependencies = Object.keys(pkg.devDependencies)
 
-    await Promise.all([
-      replaceDependencies(pkg, this.destinationPath(), pkgDependencies, 'dependencies'),
-      replaceDependencies(pkg, this.destinationPath(), pkgDevDependencies, 'devDependencies'),
-      replaceBlitzDependency(pkg, this.destinationPath(), this.options.version),
+    const dependenciesArray = await Promise.all([
+      replaceDependencies(pkg, pkgDependencies, 'dependencies'),
+      replaceDependencies(pkg, pkgDevDependencies, 'devDependencies'),
     ])
 
+    const dependencies = dependenciesArray.reduce((prev, {key, dependencies}) => {
+      prev[key] = replaceBlitzDependency(dependencies, this.options.version)
+      return prev
+    }, {} as Record<string, Record<string, string>>)
```

Comment:
Everything looks good, but could you use a for loop here instead? Reduce for object stuff is so intimidating 😅